testsuite/cluster-test/src/experiments/load_test.rs

```diff
@@ -256,30 +263,75 @@ async fn mempool_load_test(
         ));
     };
 
+    let mut bytes = 0_u64;
+    let mut tx_num = 0_u64;
     let task_start = Instant::now();
     while Instant::now().duration_since(task_start) < duration {
         let msg = libra_mempool::network::MempoolSyncMsg::BroadcastTransactionsRequest {
             request_id: lcs::to_bytes("request_id")?,
             transactions: vec![], // TODO submit actual txns
         };
         // TODO log stats for bandwidth sent to remote peer to MempoolResult
+        bytes += lcs::to_bytes(&msg)?.len() as u64;
+        tx_num += 1;
```

Comment:
there's no txn submitted, it should be request_num, see line 272

Reply:
we should implement sending txns next (the todo item), probably reuse what emit_job already has and just replace submit_transaction part

Reply:
as a fyi for the TODO^, the max number of transactions in a mempool broadcast is 100 by standard

Reply:
I think implement sending txns can be the point into next step: scale up/down the load according to system load/error rate.